discovery.go

```diff
@@ -0,0 +1,132 @@
+package viamrtsp
+
+import (
+	"bytes"
+	"encoding/xml"
+	"fmt"
+	"net"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"go.viam.com/rdk/logging"
+)
+
+const (
+    MaxUDPMessageBytesSize = 1536
+    StandardWSDiscoveryAddress = "239.255.255.250:3702"
```

Comment:
in go, if these don't need to be shared with other packages or external implementers, it's preferred to unexport them (lowercase first letter)

Reply:
Done

Reply:
Done